### PR TITLE
Added Kadane's Algorithm -- Issue No -#759

### PR DESCRIPTION
**Pull request by Hitesh4278**
Added About Kadane Algorithm
- Explained the Problem Statement
- Brute Force Approach
- How to Optimize using Kadane's algorithm
- Time Complexity
- Added in All languages
- 
Screenshot (1612)
Screenshot (1613)
Screenshot (1614)
Screenshot (1615)

### CODE REVIEW

File: dsa/Algorithms/kadanes-algorithm.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+    //if (maxi < 0) maxi = 0;
+
+    return maxi;
+}
+```
+
+## Complexity Analysis
+   #### Time Complexity : 
+    - O(n) , We are using a single loop running N times.
```
Comment (Ajay-Dhangar) [inline dsa/Algorithms/kadanes-algorithm.md:193]: replace `O(n)` to `$O(n)$` and also, `O(1)` to `$O(1)$`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update code based on suggestions

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update code based on suggestions

File: dsa/Algorithms/kadanes-algorithm.md (hunk 1 above)
Comment (Hitesh4278) [reply dsa/Algorithms/kadanes-algorithm.md]: Made the Changes. Please Look into it.

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Comment by aryan-1309** [comment]
Pls Assign this to me.

**Pull request merged**